tokio/src/runtime/coop.rs

```diff
@@ -52,10 +52,16 @@ impl Budget {
     ///
     /// Note that as more yield points are added in the ecosystem, this value
     /// will probably also have to be raised.
-    const fn initial() -> Budget {
+    pub(super) const fn initial() -> Budget {
         Budget(Some(128))
     }
 
+    cfg_unstable! {
+        pub(super) fn new(val: u8) -> Budget {
+            Budget(Some(val))
+        }
+    }
```

Comment:
I would have `initial` call this method to avoid the `cfg_unstable!`.

Reply:
Based on above 2 comments, I would propse below changes

```rust
impl Default for Budget {
    #[inline(always)]
    fn default() -> Self {
        Self::new(128)
    }
}

impl Budget {

    #[inline(always)]
    pub(super) fn new(val: u8) -> Budget {
        Budget(Some(val))
    }
}
```